src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -221,7 +221,7 @@
             <ColumnDefinition Name="Table" Width="*"/>
         </Grid.ColumnDefinitions>
         <DataGrid Grid.Column="0" 
-                  IsReadOnly="False" 
+                  IsReadOnly="True" 
```

Comment:
Based on an elusive comment in this thread:
https://stackoverflow.com/a/19808349/339720

"Note, that if IsReadOnly="False" for your DataGrid, then the first click may be stolen the selection mechanism. Set IsReadOnly="True" to fix that up." 🤦